cobra/__init__.py

```diff
@@ -9,18 +9,15 @@
 """
 # Copyright (C) 2011 Invisigoth - See LICENSE file for details
 import os
-import sys
 import json
 import time
-import errno
 import types
 import Queue
 import socket
 import struct
 import logging
 import urllib2
 import traceback
-import cPickle as pickle
```

Comment:
So this is actually used down around lines 179/198 if json or msgpack aren't selected.

Reply:
🤦 i think i searched for `cPickle` which is obviously wrong